File: NodeJS/basics/open_multiple_tabs.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch();
  const context = await browser.newContext();
  const urls = ['https://example.com', 'https://demo.opencart.com/'];

  const pages = await Promise.all(urls.map(() => context.newPage()));
  for (let i = 0; i < urls.length; i++) {
    await pages[i].goto(urls[i]);
    console.log(await pages[i].title());
  }

  await browser.close();
})();

File: Python/basics/open_multiple_tabs.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.launch()
    context = browser.new_context()
    urls = ["https://example.com", "https://demo.opencart.com/"]

    pages = [context.new_page() for _ in urls]
    for page, url in zip(pages, urls):
        page.goto(url)
        print(page.title())

    browser.close()
